web/src/app/users/UserContactMethodForm.js

```diff
@@ -33,14 +34,19 @@ export default class UserContactMethodForm extends React.PureComponent {
     onChange: () => {},
   }
 
+  state = {
+    cmAdornment: true,
+  }
+
   render() {
     const cleanValue = val => {
       val = val.replace(/[^0-9]/g, '')
 
       if (!val) {
+        this.setState({ cmAdornment: true })
```

Comment:
We should avoid using state to store calculated values, though in this case, I think we can just always render the adornment.

In the `FormField` we can use the `mapValue` prop to strip the `+` from the value since we are rendering our own.

Something like:
```js
mapValue={v => v.replace(/^\+/, '')}
```